3_COLLABORATION/parcijelni_pull_requests/sandraBanka.py

```diff
@@ -0,0 +1,122 @@
+    else:
+        novo_stanje=korisnici[br_racuna][-1]+polog
+        korisnici[br_racuna].append(novo_stanje)
+        print(f'Na br. racuna {br_racuna} se sada nalazi iznos od {novo_stanje} eura.')
+        print()
+
+def prikaz_prometa():
+    a=len(korisnici[br_racuna])-1
+    #print(a)
+    for i in range(a,2,-1):
+        print(f'\t\tStanje:\t\tUplata/isplata:\n\t\t{korisnici[br_racuna][i]} eura\t\t {korisnici[br_racuna][i]-korisnici[br_racuna][i-1]} eura')
+
+def podizanje_novca():
+        podizanje=float(input('Unesite iznos u eurima koji zelite podici s racuna: '))
+        while podizanje<0:              #da li je uneseni broj manji od 0
+            podizanje=float(input('Uneseni iznos nije pozitivan! Unesite iznos u eurima koji zelite podici s racuna: '))
+        else:
+            while korisnici[br_racuna][-1]<podizanje:                                 #provjera ima li dovoljno novca na racunu
+                podizanje=float(input('Stanje racuna je manje od zeljenog iznosa. Unesite iznos u eurima koji zelite podici s racuna: '))       
+            else:
+                podizanje_stanje=korisnici[br_racuna][-1]-podizanje
+                korisnici[br_racuna].append(podizanje_stanje)
+            print(f'Na br. racuna {br_racuna} se sada nalazi iznos od {podizanje_stanje} eura.')
+            print()
+
+while 1:
+    odg=input('Želite li se prijaviti? da/ne: ')
+    if odg=='da':
+        unos_user=input('Unesi korisničko ime administora: ')
+        logiranje(unos_user)
+        while True:
+            izbornik=int(input('\n 1. Pregled korisničkih racuna\n 2. Dodavanje novog korisnika \n 3. Pregled/uplata/isplata/prometa po racunu \n 4. Odjava \nUnesite redni broj za željenu radnju:\n'))
+            if izbornik==1:
+                pregled_korisnika()
+            elif izbornik==2:
+                novi_korisnik()
+            elif izbornik==3:
+                br_racuna=input('Unesite br. racuna: ')
+                while br_racuna not in korisnici.keys():            #provjera da li postoji navedeni br. racuna
+                    br_racuna=input('Uneseni br. racuna ne postoji. \nUnesite br. racuna: ')
+                else:
+                    while True:
+                        izbornik_m=int(input('\n 1. Pregled racuna \n 2. Prikaz prometa po racunu \n 3. Polog novca na racun \n 4. Podizanje novca s racuna \n 5. Izlaz \nUnesite redni broj za željenu radnju:\n'))
+                        if izbornik_m==1:
+                            pregled_1_korisnika()
+                        elif izbornik_m==2:
+                            prikaz_prometa()
+                        elif izbornik_m==3:
+                            polog_novca()
+                        elif izbornik_m==4:
+                            podizanje_novca()
+                        else:
+                            break
+                    else:
+                        break
+            else:
```

Comment:
Prazan input potrga loop.

` 1. Pregled korisničkih racuna
 2. Dodavanje novog korisnika 
 3. Pregled/uplata/isplata/prometa po racunu 
 4. Odjava 
Unesite redni broj za željenu radnju:

Traceback (most recent call last):
  File "/home/tom/repo/dev_26Nov2022/3_COLLABORATION/parcijelni_pull_requests/sandraBanka.py", line 92, in <module>
    izbornik=int(input('\n 1. Pregled korisničkih racuna\n 2. Dodavanje novog korisnika \n 3. Pregled/uplata/isplata/prometa po racunu \n 4. Odjava \nUnesite redni broj za željenu radnju:\n'))
ValueError: invalid literal for int() with base 10: ''`